README.md

```diff
@@ -108,9 +107,54 @@ print(response.text)
 # Interpret your results using the API - where is the space station right now ?
 ```
 
+### Finding the number of people in space
+
+Open Notify has one more API endpoint, `/astros.json`. It tells you how many people are currently in space. The format of the responses can be studied [HERE](http://open-notify.org/Open-Notify-API/People-In-Space/).
+
+Read the above documentation and perform the following tasks:
+
+* Get the response from astros.json endpoint
+* Count how many people are currently in space
+* List the names of people currently in space.
+
+
+```python
+# Your Code Here
+```
+
+
+```python
+# Interpret the Results - How many people are in space and what are their names 
+```
+
+
+```python
+# __SOLUTION__ 
+# Get the response from the API endpoint.
+response = requests.get("http://api.open-notify.org/astros.json")
+data = response.json()
+
+# 9 people are currently in space.
+print(data["number"])
+print(dict(data))
+
+```
+
+    10
+    {'message': 'success', 'people': [{'name': 'Zhai Zhigang', 'craft': 'Shenzhou 13'}, {'name': 'Wang Yaping', 'craft': 'Shenzhou 13'}, {'name': 'Ye Guangfu', 'craft': 'Shenzhou 13'}, {'name': 'Raja Chari', 'craft': 'ISS'}, {'name': 'Tom Marshburn', 'craft': 'ISS'}, {'name': 'Kayla Barron', 'craft': 'ISS'}, {'name': 'Matthias Maurer', 'craft': 'ISS'}, {'name': 'Oleg Artemyev', 'craft': 'ISS'}, {'name': 'Denis Matveev', 'craft': 'ISS'}, {'name': 'Sergey Korsakov.', 'craft': 'ISS'}], 'number': 10}
+
+
+
+```python
+# __SOLUTION__ 
+# Interpret the Results - How many people are in space and what are their names 
+```
+
+## Level Up (Optional) - Investigating other API endpoints
+
 ### Check the next pass of International space station for a given location
 
-Let's repeat the above for the second endpoint `iss-pass.json`. This end point is used to query the next pass of the space station on a given location. Let's just run as above and record your observations.
+Let's repeat the exercise for the another end point `iss-pass.json`. This end point is used to query the next pass of the space station on a given location.
```

Comment:
Currently says "for the another", let's just make it "for another"

Let's also make it "endpoint" instead of "end point" in both places in this line